doc/user-guide/password.md

```diff
@@ -21,6 +21,77 @@ To enable it, simply turn the option on and set a user and a password:
 Your password isn't visible after deployment so write it down!
 ```
 
+## Basic Password Protection with the CLI
+
+Ploomber Cloud allows you to add authentication for specific features of your application using the CLI. This provides an additional layer of security for sensitive parts of your project.
+
+### Adding Authentication
+
+To add authentication for a specific feature, use the following command:
+
+```bash
+ploomber-cloud auth --add --feature <feature_name>
+```
+
+Where `<feature_name>` is one of the supported features compatible with authentication.
+
+If you don't specify a feature, you will be prompted to select one from the available options.
+
+### Options
+
+- `--overwrite`: Use this flag to overwrite existing authentication fields if they already exist.
+
+### Removing Authentication
+
+To remove authentication for a specific feature, use:
+
+```bash
+ploomber-cloud auth --remove --feature <feature_name>
+```
+
+### Important Notes
+
+1. You can only add or remove authentication at a time, not both simultaneously.
+2. At least one action (add or remove) must be specified.
+3. The `--overwrite` flag can only be used when adding authentication, not when removing.
+
+### Authentication Process
+
+When adding authentication:
+
+1. You will be prompted to enter a username and password for the selected feature.
+2. If credentials already exist in the `.env` file and `--overwrite` is not used, those credentials will be reused.
```

Comment:
are they stored in an .env file? I did some test and I saw the credentials were stored in the ploomber-cloud.json

Reply:
> are they stored in an .env file? I did some test and I saw the credentials were stored in the ploomber-cloud.json

They are also being stored in a .env file in case the user wanted to provide the values from there (similar to how the auth0 template works)

Reply:
then the documentation is confusing. it only mentions the `.env` file, but it doesn't mention `ploomber-cloud.json`. are they stored in both? or only one? it should be clear from the docs where they're stored and why

Reply:
> then the documentation is confusing. it only mentions the `.env` file, but it doesn't mention `ploomber-cloud.json`. are they stored in both? or only one? it should be clear from the docs where they're stored and why

[Added ploomber-cloud.json mention](https://github.com/ploomber/doc/pull/283/files#diff-ee5b667893ae9f81da782a6407f629f366501a0e2688f6ba2477c70cd8963a5cR72)